kolibri/auth/constants/facility_configuration_presets.json

```diff
@@ -24,6 +23,7 @@
   },
   "informal": {
     "name": "Informal and personal use",
+    "default": true,
```

Comment:
seems to be a discrepancy here. This says that 'informal' is the default, but the onboarding wizard says that 'self-managed' is the default

(side note - I think we should avoid the informal/nonformal distinction in code for the same reason we avoid it in the UI: it's confusing as hell :) )